ironfish/src/rpc/routes/router.ts

```diff
@@ -142,4 +142,10 @@ class Routes {
   }
 }
 
+export function isFullNode(x: unknown, message?: string): asserts x is IronfishNode {
```

Comment:
Is this used right now? Should we use this in all the assertions in the RPC handlers?

Reply:
Not used, removing